### PR TITLE
Some performance improvement

### PR DESCRIPTION
**Pull request by wtdcode**
This PR is authored by @radare and tested against https://github.com/athre0z/disas-bench and valgrind.

### CODE REVIEW

**Comment by kabeor** [comment]
cool, we are faster now!

File: MCInst.c [hunk 1]
```diff
@@ -17,12 +17,17 @@
 
 void MCInst_Init(MCInst *inst)
 {
+#if 0
```
Comment (tmfink) [inline MCInst.c:20]: If this is not needed, then if should just be deleted.

File: cs.c [hunk 2]
```diff
@@ -554,6 +554,15 @@ static void fill_insn(struct cs_struct *handle, cs_insn *insn, char *buffer, MCI
...
 #ifndef CAPSTONE_DIET
+	char *tab = strchr (buffer, '\t');
+	if (tab) {
+		*tab = 0;
+	}
+	strncpy (insn->mnemonic, buffer, sizeof (insn->mnemonic) - 1);
+	if (tab) {
+		strcpy (insn->op_str, tab + 1);
+	}
+#if 0
```
Comment (tmfink) [inline cs.c:565]: dead code should just be removed

File: cs.c [hunk 3]
```diff
@@ -588,6 +598,7 @@ static void fill_insn(struct cs_struct *handle, cs_insn *insn, char *buffer, MCI
 		}
 	}
 
+#if 0
```
Comment (tmfink) [inline cs.c:601]: dead code should just be removed

**Comment by kabeor** [comment]
@tmfink cleaned up, thank you.

**Pull request merged**